src/xrdreq/QservRequest.cc

```diff
@@ -118,6 +134,7 @@ bool QservRequest::ProcessResponse(const XrdSsiErrInfo& eInfo, const XrdSsiRespI
             // WARNING: This has to be the last call as the object may get deleted
             //          downstream.
             onError("QservRequest::ProcessResponse  ** ERROR ** unexpected response type: " + responseType);
+            _self = nullptr;
```

Comment:
Good idea ... except the right way of doing this is:
```
auto localSelfKeepAlive = std::move(_self);
```
Otherwise, _self will never get deleted if any crash happens before `_self = nullptr;`